packages/base/src/index.js

```diff
@@ -4,82 +4,115 @@ module.exports = {
     'shared-node-browser': true,
   },
 
-  plugins: ['import'],
+  plugins: ['import', 'prettier'],
 
-  extends: ['eslint:recommended'],
+  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
 
   rules: {
-    'accessor-pairs': 'error',
-    'array-bracket-spacing': ['error', 'never'],
-    'array-callback-return': 'error',
-    'arrow-parens': 'error',
-    'arrow-spacing': [
+    'prettier/prettier': [
       'error',
       {
-        before: true,
-        after: true,
+        singleQuote: true,
+        trailingComma: 'all',
+        quoteProps: 'consistent',
+      },
+      {
+        // Allow consumers to override this prettier config.
+        usePrettierrc: true,
       },
     ],
-    'block-scoped-var': 'error',
-    'block-spacing': ['error', 'always'],
-    'brace-style': 'error',
-    'camelcase': [
+
+    // In order to match the prettier spec, you have to enable lines around
+    // comments before and after blocks, objects, and arrays.
+    // https://github.com/prettier/eslint-config-prettier#lines-around-comment
+    'lines-around-comment': [
       'error',
       {
-        properties: 'never',
-        allow: ['^UNSAFE_'],
+        beforeBlockComment: true,
+        afterLineComment: false,
+        allowBlockStart: true,
+        allowBlockEnd: true,
+        allowObjectStart: true,
+        allowObjectEnd: true,
+        allowArrayStart: true,
+        allowArrayEnd: true,
       },
     ],
-    'comma-dangle': ['error', 'always-multiline'],
-    'comma-spacing': [
+
+    // Prettier has some opinions on mixed-operators, and there is ongoing work
+    // to make the output code clear. It is better today then it was when the first
+    // PR to add prettier. That being said, the workaround for keeping this rule enabled
+    // requires breaking parts of operations into different variables -- which I believe
+    // to be worse. https://github.com/prettier/eslint-config-prettier#no-mixed-operators
+    'no-mixed-operators': 'off',
```

Comment:
these are turned off by the recommended settings from prettier.

Reply:
I like that the comments function as documentation, so I'll leave those in place.